src/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface Category {
+    id: number;
+    title: string;
+}
+
+export interface Question {
+    id: number;
+    title: string;
+    body: string;
+    format: string;
+    releaseDate: string;
+    expirationDate: string;
+    difficulty: string;
```

Comment:
Apart from some minor naming changes which can be taken care of easily, LGTM

Reply:
These are the names we currently have for the columns on the backend. Which names do you think should be changed?